recipe/meta.yaml

```diff
@@ -0,0 +1,195 @@
+{% set name = "torchaudio" %}
+{% set version = "2.1.0" %}
+{% set run_all_unittests = False %}
+# Torchaudio and PyTorch are tightly coupled in terms of versions.
+# see compatibility matrix in https://github.com/pytorch/audio/blob/v2.1.2/docs/source/installation.rst#compatibility-matrix
+{% set compatible_pytorch = "2.1.0" %}
```

Comment:
```suggestion
```
It's annoying to have a dependency version defined outside the dependencies block. I requires us to scroll and search for that variable.

Reply:
that was me fwiw, to give attention to the compatibility pairing. having the version lower as you say sounds good but I think the comment should stay so engineers working on the package in future don't have to re-discover this information and possibly miss it.

Reply:
addressed.